src/app/rosetta/lib/operation_types.ml

```diff
@@ -0,0 +1,38 @@
+open Core_kernel
+
+type t =
+  [ `Fee_payer_dec
+  | `Fee_creator_inc
+  | `Coinbase_inc
+  | `Account_creation_fee
```

Comment:
As discussed OOB, I think this should be split into cases for the fee-payer paying  (transactions) and the receiver paying (create token/account)